tensorflow_addons/utils/resource_loader.py

```diff
@@ -20,8 +20,8 @@
 
 import tensorflow as tf
 
-MIN_TF_VERSION_FOR_ABI_COMPATIBILITY = "2.2.0"
-MAX_TF_VERSION_FOR_ABI_COMPATIBILITY = "2.3.0"
+MIN_TF_VERSION_FOR_ABI_COMPATIBILITY = "2.3.0"
+MAX_TF_VERSION_FOR_ABI_COMPATIBILITY = "2.4.0"
```

Comment:
Is 2.4.0 ok?

Reply:
Not okay, so we check the version via `min_version <= version < max_version`.

https://github.com/tensorflow/addons/blob/master/tensorflow_addons/utils/resource_loader.py#L115

Reply:
Do you remember why we had that? It Is a little bit less intuitive that the MAX compatibility It is excluded

Reply:
Actually, I don't know about that too much. I guess it is because

```python
>>> LooseVersion("2.3.1") <= LooseVersion("2.3")
False
```

Reply:
Yeah strange variable name that should be updated. Also we should update the release_version.sh to update resource_loader

Reply:
Ok but we I see we use extended version with the final `.0`